src/Builder/MySQLBuilder.java

```diff
@@ -0,0 +1,31 @@
+package Builder;
+
+public class MySQLBuilder {
+
+	private BuilderConnection con;
+	
+	public MySQLBuilder() {
+		con = new BuilderConnection();
+	}
+
+	//@Override
+	public void buildUser() {
+		con.setUser("root");
```

Comment:
the builder should receive the information via parameters. for example:

new Builder().withField1("field1").withField2("field2).build()